docs/build-framework/switches/host-docker.md

```
@@ -0,0 +1,388 @@
+---
+seo_title: "Armbian build host & Docker switches"
+description: "Configure the Armbian build host and Docker container: base image, auto-pull, pruning, OCI cache, HTTP proxies, download mirrors and the apt cache."
+---
+
+# Build host & Docker
+
+Control the build host, the Docker build container, network proxies and mirrors, and the apt package cache.
+
+## Build host & Docker
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)